docs/pages/eas/workflows/syntax.mdx

```diff
@@ -4,17 +4,33 @@ description: Reference guide for the EAS Workflows configuration file syntax.
 maxHeadingDepth: 5
 ---
 
+{/* vale off */}
+
 import { GithubIcon } from '@expo/styleguide-icons/custom/GithubIcon';
 
 import { BoxLink } from '~/ui/components/BoxLink';
+import { FileTree } from '~/ui/components/FileTree';
 
 A workflow is a configurable automated process made up of one or more jobs. You must create a YAML file to define your workflow configuration.
 
-## YAML syntax for workflows
+To get started with workflows, see [Get Started with EAS Workflows](/eas/workflows/get-started) or see [Examples](/eas/workflows/examples) for complete workflow configurations.
+
+## Workflow files
 
 Workflow files use YAML syntax and must have either a `.yml` or `.yaml` file extension. If you're new to YAML and want to learn more, see [Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml/).
 
-Workflow files are located in the `.eas/workflows` directory in your project, and should be at the same level as your [**eas.json**](/eas/introduction/eas-json) file.
+Workflow files are located in the `.eas/workflows` directory in your project. The `.eas` directory should be at the same level as your [**eas.json**](/eas/introduction/eas-json) file.
```

Comment:
```suggestion
Workflow files are located in the **.eas/workflows** directory in your project. The **.eas** directory should be at the same level as your [**eas.json**](/build/eas-json/) file.
```

- Directory and file names are bold as per our writing style guide
- Fix internal link